benches/common/mod.rs

```diff
@@ -1,9 +1,10 @@
-use init_logger;
+extern crate test;
+
+use super::{init_logger, util::with_redirected_stderr};
+use log::*;
 use std::panic;
 use test::Bencher;
```

Comment:
Some of these changes now don't pass a `cargo fmt` cleanly.

Reply:
I thought I noticed that the original code wasn't fmt-clean, so I didn't pay too much attention to it. I'll setup a git hook to check for fmt.